test/unit/storage/DataAccessorBasedStore.test.ts

```diff
@@ -424,10 +442,13 @@ describe('A DataAccessorBasedStore', (): void => {
       );
     });
 
-    it('errors when trying to create a container with non-RDF data.', async(): Promise<void> => {
+    it('succeeds when trying to create a container with non-RDF data +' +
```

Comment:
Think we can just drop this test since it's identical to the "ignores content" test above right? Except for the content-type but that doesn't matter any more.